logging/api_reference/logging-5-8-reference.adoc

```diff
@@ -1,7 +1,1568 @@
+
+Type:: array
+
+=== .status.logStore.elasticsearchStatus[].statefulSets[]
+
+Type:: array
+
+=== .status.visualization
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|kibanaStatus|array|  *(optional)*
+|======================
+
+=== .status.visualization.kibanaStatus[]
+
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|clusterCondition|object|  *(optional)*
+|deployment|string|  *(optional)*
+|pods|string|  *(optional)* The status for each of the Kibana pods for the Visualization component
+|replicaSets|array|  *(optional)*
+|replicas|int|  *(optional)*
+|======================
+
+=== .status.visualization.kibanaStatus[].clusterCondition
+
+Type:: object
+
+=== .status.visualization.kibanaStatus[].replicaSets[]
+
+Type:: array
+
+[id="logging-5-8-reference-LogFileMetricExporter"]
+== LogFileMetricExporter
+
+A Log File Metric Exporter instance. LogFileMetricExporter is the Schema for the logFileMetricExporters API
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|spec|object|
+|status|object|
+|======================
+
+=== .spec
+
+LogFileMetricExporterSpec defines the desired state of LogFileMetricExporter
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|nodeSelector|object|  *(optional)* Define which Nodes the Pods are scheduled on.
+|resources|object|  *(optional)* The resource requirements for the LogFileMetricExporter
+|tolerations|array|  *(optional)* Define the tolerations the Pods will accept
+|======================
+
+=== .spec.nodeSelector
+
+Type:: object
+
+=== .spec.resources
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|claims|array|  *(optional)* Claims lists the names of resources, defined in spec.resourceClaims,
+|limits|object|  *(optional)* Limits describes the maximum amount of compute resources allowed.
+|requests|object|  *(optional)* Requests describes the minimum amount of compute resources required.
+|======================
+
+=== .spec.resources.claims[]
+
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|name|string|  Name must match the name of one entry in pod.spec.resourceClaims of
+|======================
+
+=== .spec.resources.limits
+
+Type:: object
+
+=== .spec.resources.requests
+
+Type:: object
+
+=== .spec.tolerations[]
+
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```

Comment:
cut off

Reply:
Think I've found a fix for this now 🥳 gonna update the API templates

Reply:
oh, good!